### ISSUE TITLE
type not used precompilation warning

### ISSUE THREAD

**Issue by CarloLucibello** [PhilipVinc/TensorBoardLogger.jl#121]
```
WARNING: method definition for preprocess at 
/Users/carlo/.julia/packages/TensorBoardLogger/Ukp9t/src/logger_dispatch.jl:73 
declares type variable T but does not use it.
```

**Comment by CarloLucibello** [comment]
fixed in #122

**Comment by CarloLucibello** [comment]
@oxinabox can we get a new release?
